lms/djangoapps/badges/migrations/0004_badgeclass_badgr_server_slug.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('badges', '0003_schema__add_event_configuration'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='badgeclass',
+            name='badgr_server_slug',
+            field=models.SlugField(blank=True, default=b'', max_length=255),
```

Comment:
The default in this migration file should not be a `binary` string, right? Please update so the migration is consistent with model update.